DatabaseBuddy.Test/Core/Extender/TestOfListExtender.cs

```diff
@@ -0,0 +1,62 @@
+﻿using DatabaseBuddy.Core.Extender;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DatabaseBuddy.Test.Core.Extender
+{
+  [TestClass]
+  public class TestOfListExtender
```

Comment:
Please insert checks to ensure that the correct entries have been removed. Give the list entries any values which can be checked afterwards
Please insert checks to ensure that the correct entries have been removed.